### PR TITLE
Starting cleanup for deploy-pipeline overview

### PR DESCRIPTION
**Pull request by stephendonner**
Just getting started, but wanted to reflect current reality - more to come.

@davehunt @chartjes @rpappalax - nothing too specific/controversial, I hope?  r?

### CODE REVIEW

File: reference/ci.rst [hunk 1]
```diff
@@ -61,12 +61,8 @@ Ops-QA Pipeline
...
-3. Create a Jenkins job with the following syntax: **project.test_env** (e.g. **kinto.stage**), using the ``Pipeline from SCM`` option, and pointing to the Jenkinsfile
-4. Once your project is set up properly (runs, and hopefully passes):
-5. File a bug (example: `bug 1384404 <https://bugzilla.mozilla.org/show_bug.cgi?id=1384404>`_), in the Cloud Services product, FXTest-infra component, requesting Ops enable your jobs in their pipeline
-6. Next, from Ops' side, there is a `qaTest.groovy file <https://github.com/mozilla-services/cloudops-deployment/blob/c6a09fa1a62d1cddf3a3b560e92aca55a497d0d4/libs/pipeline/vars/qaTest.groovy#L13>`_ which authenticates with QA (prod) Jenkins, and will run /job/${project}.${envName}
+1. A suggestion is to have your project build and run tests in Jenkins, from a Docker image
+2. Create a Jenkins job with the following syntax: **project.test_env** (e.g. **kinto.stage**), using the ``Pipeline from SCM`` option, and pointing to the Jenkinsfile
+3. Once your project is set up properly (runs, and hopefully passes):
```
Comment (rpappalax) [inline reference/ci.rst]: or perhaps more simply:  "Once your project runs and passes, ..."

**Comment by rpappalax** [comment]
other than small nit, 👍

File: reference/ci.rst (hunk 1 above)
Comment (stephendonner) [reply reference/ci.rst]: :100: - yes, thanks; will make that change.

**Comment by stephendonner** [comment]
Imma merge - it's basically a Wiki backed by SCM, and I'd like to continue to iterate.

**Pull request merged**